index.js

```diff
@@ -72,6 +72,23 @@ module.exports = {
     "simple-import-sort/imports": "error",
     "simple-import-sort/exports": "error",
     "sort-destructure-keys/sort-destructure-keys": "error",
+    /**
+     * List of sonarjs rules we turned off after release of eslint-plugin-sonarjs v2
+     * Some rules are too restrictive for common use cases,
+     * and others are already covered by other plugins used here.
+     */
+    "sonarjs/different-types-comparison": "off",
+    "sonarjs/function-return-type": "off",
+    "sonarjs/no-clear-text-protocols": "warn",
+    "sonarjs/no-ignored-exceptions": "off",
```

Comment:
`// We may want to catch errors but not use the error object directly, just trigger error handling fallbacks within the catch block.`